Vic2ToHoI4/Source/V2World/World.cpp

```diff
@@ -26,44 +26,40 @@ Vic2::World::World(const mappers::ProvinceMapper& provinceMapper, const Configur
 	inventions theInventions(theConfiguration);
 
 	std::vector<int> GPIndexes;
-	registerKeyword(std::regex("great_nations"), [&GPIndexes, this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("great_nations", [&GPIndexes, this](const std::string& unused, std::istream& theStream) {
 		const commonItems::intList indexList(theStream);
 		GPIndexes = indexList.getInts();
 	});
 
-	registerKeyword(std::regex("\\d+"), [this, &theIssues](const std::string& provinceID, std::istream& theStream) {
+	registerRegex(R"(\d+)", [this, &theIssues](const std::string& provinceID, std::istream& theStream) {
 		provinces[stoi(provinceID)] = new Province(provinceID, theStream, theIssues);
 	});
 
 	std::vector<std::string> tagsInOrder;
 	tagsInOrder.push_back(""); // REB (first country is index 1
-	registerKeyword(std::regex("[A-Z]{3}"),
+	registerRegex("[A-Z]{3}|[A-Z][0-9]{2}",
```

Comment:
I really thought my suggestion [A-Z][A-Z0-9]{2} was shorter, but sure, this works too.

Reply:
Oh, so it is. Replacing mine for yours.